test/e2e/e2e_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	_ "github.com/submariner-io/shipyard/test/e2e/example"
-	_ "github.com/submariner-io/submariner/test/e2e/framework"
+	_ "github.com/submariner-io/shipyard/test/e2e/framework"
```

Comment:
This was an oversight - I did not intend any code dependency on submariner.  Can you also remove github.com/submariner-io/submariner from go.mod?

Reply:
I thought of that too, but `go.mod` doesn’t mention submariner.

Reply:
yeah - it was in mine local go.mod - that's wierd.

Reply:
ahh - it gets added when you build due to this import 😄